### PR TITLE
Fix scalacheck `Gen` for Avro `fixed` type

### PR DESCRIPTION
**Pull request by kanterov**

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -11,3 +11,9 @@ addSbtPlugin("org.xerial.sbt" % "sbt-sonatype" % "1.1")
 libraryDependencies ++= Seq(
   "com.github.os72" % "protoc-jar" % "3.3.0.1"
 )
+
+// sbt-avro-1-7 uses avro 1.7, so we bump it to 1.8.1
+libraryDependencies ++= Seq(
```
Comment (nevillelyh) [inline project/plugins.sbt]: Should use `version in avroConfig` instead?

Comment (kanterov) [inline project/plugins.sbt]: Hm.. doesn't matter, works in both ways

Comment (kanterov) [inline project/plugins.sbt]: Hm.. but it isn't there, this is inner project that doesn't know anything about outer

Comment (kanterov) [inline project/plugins.sbt]: It work work if `sbt-avro` was in `project/project/plugins.sbt`

Comment (kanterov) [inline project/plugins.sbt]: I guess it work work when `sbt-avro-1-8` will be released

**Comment by kanterov** [comment]
@nevillelyh any chance for release? :)

**Comment by nevillelyh** [comment]
Just released!

**Pull request merged**